[2025-04-18T19:31:24.553+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion.local_data_to_gcs_task manual__2025-04-18T19:23:37.431616+00:00 [queued]>
[2025-04-18T19:31:24.579+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion.local_data_to_gcs_task manual__2025-04-18T19:23:37.431616+00:00 [queued]>
[2025-04-18T19:31:24.582+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-04-18T19:31:24.645+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): local_data_to_gcs_task> on 2025-04-18 19:23:37.431616+00:00
[2025-04-18T19:31:24.653+0000] {standard_task_runner.py:57} INFO - Started process 363 to run task
[2025-04-18T19:31:24.670+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion', 'local_data_to_gcs_task', 'manual__2025-04-18T19:23:37.431616+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmpd30z5l49']
[2025-04-18T19:31:24.675+0000] {standard_task_runner.py:85} INFO - Job 67: Subtask local_data_to_gcs_task
[2025-04-18T19:31:24.701+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-04-18T19:31:24.866+0000] {task_command.py:416} INFO - Running <TaskInstance: data_ingestion.local_data_to_gcs_task manual__2025-04-18T19:23:37.431616+00:00 [running]> on host e929439072d8
[2025-04-18T19:31:25.096+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion' AIRFLOW_CTX_TASK_ID='local_data_to_gcs_task' AIRFLOW_CTX_EXECUTION_DATE='2025-04-18T19:23:37.431616+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-18T19:23:37.431616+00:00'
[2025-04-18T19:31:36.035+0000] {data_ingestion.py:105} INFO - File /opt/***/data/parquet/customers.parquet uploaded to customers.parquet.
[2025-04-18T19:31:36.489+0000] {data_ingestion.py:105} INFO - File /opt/***/data/parquet/products.parquet uploaded to products.parquet.
[2025-04-18T19:32:12.407+0000] {data_ingestion.py:105} INFO - File /opt/***/data/parquet/transactions.parquet uploaded to transactions.parquet.
[2025-04-18T19:32:12.410+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-04-18T19:32:12.482+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=data_ingestion, task_id=local_data_to_gcs_task, execution_date=20250418T192337, start_date=20250418T193124, end_date=20250418T193212
[2025-04-18T19:32:12.753+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-18T19:32:13.122+0000] {taskinstance.py:2778} INFO - 3 downstream tasks scheduled from follow-on schedule check
